app/oauth-start.sh

```diff
@@ -1,24 +1,28 @@
 #!/bin/sh
 NGINX_PID="/var/run/nginx.pid"    # /   (root directory)
-APP="oauth_daemon.py"
-
-NGINX_CONF="/etc/nginx/fabric_nginx_$CONTAINER_ENGINE.conf";
+NGINX_CONF="";
 NGINX="nginx";
-
-python /usr/src/app/$APP &
-
-if [ "$NETWORK" = "fabric" ]
-then
-    echo fabric configuration set;
-fi
+APP="oauth_daemon.py"
 
 if [ "$DEBUG" = "true" ]
 then
     NGINX="nginx-debug";
     echo System is set to DEBUG;
 fi
 
-$NGINX -c "$NGINX_CONF" -g "pid $NGINX_PID;" &
+case "$NETWORK" in
+    fabric)
+        NGINX_CONF="/etc/nginx/fabric_nginx_$CONTAINER_ENGINE.conf"
+        echo 'Fabric configuration set'
+        $NGINX -c "$NGINX_CONF" -g "pid $NGINX_PID;" &
```

Comment:
I don't think this will work.

If the NETWORK is set to something other than fabric, the command to start NGINX will never be executed.

Reply:
Disregard this comment. I get now that we only want NGINX to start when using the fabric model.

Does the logic also check that NGINX is running if using the fabric model?